### DISCUSSION TITLE
Average Person less risk than eating out once

### DISCUSSION THREAD

**Discussion by mariaj1990** [microCOVID/microCOVID#1432] [Q&A]
I understand how the risks are calculated. From a common stance standpoint, I do not think it makes since that eating out once (1,100 microcovid) is a higher risk than the average person in my area (700 microcovid). Nearly everyone in the area has stopped taking precautions, and I expect that the average person eats out or attends a similar gathering at least once a week. 

Would love to hear what everyone else thinks of this. It seems to have become an issue since the omicron update.

**Comment by zerotrickpony** [comment]
I hear you, and I've had similar thoughts about the way the tool models the "went to a bar" person risk. A growing portion of these behaviors' risks are "priced in" to the local prevalence, so as they get more typical they should also be closer to the mean. By how much, is the less clear question.

That said, I'd suggest not worrying too much about the comparison between the "person risk" score to the "activity risk" of the scenario calculation, it's a bit apples and oranges. Person risk models "has covid", which is about snapshotting the person or population at a moment in time. That's mainly by the estimate of local prevalence, unless you know a lot about the specific person, which for strangers at a restaurant you definitely don't. Activity risk models "you GET covid" which has a significant time factor (how long are you unmasked at the restaurant) and multiplicative factor (how many diners are you swapping air with).

For the specific question of indoor dining comparison, a few points:

#0, in your area's case I think 700 and 1100 are pretty close, well within error for a scenario calculation. So you might think of these two numbers as "basically the same" and not worry too much about the within-error difference.

#1, you might be wrong about the prevalence of the dining behavior in your area --- a lot of people with kids or low income don't eat out much, for example. So your behavior might be more outgoing vs the average than you think. Then again, restaurants may not be driving infections in your area as much as schools or local industry... who knows what's inside that average.

#2, an individual's "true" person risk spikes for the couple days after the restaurant activity, but then declines as the evidence accumulates that they didn't get covid from it. The average of that curve will be some number below the peak.

Hope these thoughts help!

**Reply by lazerwalker** [reply]
Sorry for the post necro, but I'd be curious to hear if your thoughts on the "went to a bar" risk profile have shifted at all in the months since. As it feels like "going to a bar/restaurant in the last 10 days" has normalized more and more, it feels like this risk is more "priced in" to local prevalence than it was when this thread was raised in April. I think having this risk profile as an option for contacts still has value, but (not knowing how the original calculations of "this is a 20x multiplier over average risk" were calculated) I wonder if it's currently somewhat overestimating risk, or how we'd even assess if it was.

**Reply by apiology** [reply]
> (not knowing how the original calculations of "this is a 20x multiplier over average risk" were calculated

@lazerwalker: Here's how I tried to answer that question:

1) Looked through the UI.  The pop-up from the icon says:

> An "average risk person in your area" is based on the prevalence of COVID in your geographic area, as explained in the [Basic Method](https://www.microcovid.org/paper/7-basic-method) from the white paper.

> We have modeled several additional Person Risk examples using the [Advanced Method](https://www.microcovid.org/paper/9-advanced-method) from the white paper. Select the one that most closely matches the person or people you are planning to interact with. We suggest you "round up" if you are uncertain.

<img width="200" alt="image" src="https://user-images.githubusercontent.com/3681194/209134525-a9c8a39f-0791-4ef6-9683-952605bad5c8.png">

So I went to the white paper.  The page linked didn't reference those particular scenarios, but in the next section -   https://www.microcovid.org/paper/10-example-person-risk - there's a sentence that says:

> In our calculator we’ve provided some Person Risk Profiles that were made using the Advanced M
ethod. You can use them as a starting point to create your own calculations.

Hmmm, not useful as it could be.  Let's use the history on the codebase and see if that gives any pointers.

The search box gives these results: https://github.com/microCOVID/microCOVID/search?q=Went+to+a+bar+in+the+last+10+days

<img width="200" alt="image" src="https://user-images.githubusercontent.com/3681194/209135256-c2764487-daa0-4d76-b3af-544d01eb8150.png">

The first hit looks interesting - there was a blog post relating to this!  A little stumbling around the site led to this URL: https://www.microcovid.org/tracker/basics

<img width="586" alt="image" src="https://user-images.githubusercontent.com/3681194/209135718-cfb8cabf-dad8-4765-a50c-343e51cf1596.png">

> Went to a bar / party in the last 10 days

> A known person who spends 6 hours per week in settings where people are close together, unmasked and talking loudly.

This is a more specific description of the experience than is given in the calculator.

**Reply by apiology** [reply]
You might be able to find more by using the 'blame' function in GitHub from there from the hits on the search results.
